src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -288,6 +288,7 @@ export class SelectBoxList implements ISelectBoxDelegate, IListVirtualDelegate<I
 		}
 
 		this.selectElement.selectedIndex = this.selected;
+		this.selectElement.title = this.options[this.selected] && this.options[this.selected].text ? this.options[this.selected].text : '';
```

Comment:
We need a `// {{SQL CARBON EDIT}}` tag here at a minimum or it'll be lost in next merge. Have you checked with @anthonydresser or others on if there's a different way to resolve that doesn't make us do VSCode changes?

Reply:
Going back to the conversation here #4542 the standard for inputs when it comes to title and aria-labels on the web is that they don't describe the currently selected value, but rather what the input is for. I think we should stick to this unless we get informed otherwise by the accessibility team.

Reply:
@kevcunnane, Anthony and I looked this. We found out it is a regression in VS code. May be caused by https://github.com/microsoft/vscode/commit/6bab1c3ee1f5c580ac14579be88b2da09064cd57. 

We thought about the solution and decided to open a issue and see if VS code team has any plan for it. https://github.com/microsoft/vscode/issues/73844

Before we resolve the tooltip issue, it is better to keep the current behavior rather than cutting off the text.

Reply:
How critical is this fix? Vscode said they will look into it. Can we just wait for them to fix it?